Cargo.toml

```diff
@@ -33,7 +33,7 @@ raw-window-handle = "0.3"
 bitflags = "1"
 
 [dev-dependencies]
-image = "0.23"
+image = "0.23.12"
```

Comment:
Doesn't build with earlier version of image afaics.

Reply:
This is one of the reasons why I would **always** encourage specifying the full versions. Only because it builds with 0.23.X, doesn't mean it builds with 0.23.0. So I'd rather have it a bit higher than necessary than too low.